src/functions/html.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import defineFunction, {ordargument} from "../defineFunction";
+import buildCommon from "../buildCommon";
+import {assertNodeType} from "../parseNode";
+
+import * as html from "../buildHTML";
+import * as mml from "../buildMathML";
+
+defineFunction({
+    type: "html",
+    names: ["\\htmlClass", "\\htmlId", "\\htmlStyle", "\\htmlData"],
```

Comment:
In MathJax, they are `\class`, `\cssId`, and `\style`, respectively, (they don't have one that can set data attributes) but I think their names are too generic and we have/will have multiple renderers, so I named them with the prefix `html`.

Reply:
`\htmlData` takes key/value pairs as the first argument and sets data attributes. Inspired by #1698.

Reply:
I like the prefixing.  Thanks for adding `\htmlData` as well.